openlibrary/templates/books/author-autocomplete.html

```diff
@@ -38,28 +38,8 @@
         <a href="javascript:;" class="remove red plain" title="$_('Remove this author')">[&times;]</a>&nbsp;
         <a href="javascript:;" class="add">$_("Add another author?")</a>
     </div>
-
-<div id="authors">
```

Comment:
MINOR UI REGRESSION:

Removing this ID  causes the cross icon to jump to a new line for me. Let's make sure the rule `#authors div, div.lists` still applies - potentially by adding `multi-input-autocomplete--author` to static/css/page-user.less for now.


<img width="745" alt="Screen Shot 2021-05-23 at 10 25 35 AM" src="https://user-images.githubusercontent.com/148752/119270371-38b62a00-bbb1-11eb-9645-5060b64ac776.png">